CONTRIBUTING.md

```diff
@@ -0,0 +1,51 @@
+# Contributing
+We are glad that you are interested in contributing to the Plasma Engine! We are happy to accept contributions, including:
```

Comment:
This seems repetitive.  Perhaps "We are glad that you are interested in contributing to the Plasma Engine! We happily accept contributions, including"